Client/Frontend/Browser/BrowserViewController/BrowserViewController+WebViewDelegates.swift

```diff
@@ -398,6 +398,15 @@ extension BrowserViewController: WKNavigationDelegate {
     func webView(_ webView: WKWebView, didFinish navigation: WKNavigation!) {
         if let tab = tabManager[webView] {
             navigateInTab(tab: tab, to: navigation)
+
+            // If this tab had previously crashed, wait 5 seconds before resetting
+            // the consecutive crash counter. This allows a successful webpage load
+            // without a crash to break out of a content process crash loop.
+            if tab.consecutiveCrashes > 0 {
+                DispatchQueue.main.asyncAfter(deadline: .now() + .seconds(5)) {
+                    tab.consecutiveCrashes = 0
```

Comment:
maybe check if tab.consecutiveCrashes is unchanged before setting to zero?

Reply:
👍